### DISCUSSION TITLE
Option to have the floating toolbar on the upper right or left?

### DISCUSSION THREAD

**Discussion by daxmonson** [chrisgurney/obsidian-note-toolbar#106] [Q&A] [answered]
Great project and so on top of communicating new features here.  Maybe I'm missing it, but is there an option to have the floating toolbar (circle with ellipsis inside) float on the upper left or right near properties _on Mobile_? I tried with the style settings plugin/padding, but can't seem to move the location so I'm assuming it's a no go? 

Reasoning is, when I'm using mobile and it's floating low by the mobile toolbar, instead of upper/tab bar, often it falls out of line of sight. Having it upper would be a nice visual reminder to use it. Have long used Commander as well and it's taking space up top so no room to have it horizontal. 

Thanks!

**Comment by chrisgurney** [comment, marked as answer]
Hi @daxmonson , thanks for the kind words!

Try: _Style Settings > Floating button > Sizing and Spacing > Button Y position on mobile_ → I set it to `20` to achieve the below (iPhone 15). You may have to make adjustments based on your screen height.

- Generally the smaller the number the higher it will be placed.
- (Note that the menu will still appear from the bottom when the button's tapped, as that's how mobile menus work in Obsidian.)

<img src="https://github.com/user-attachments/assets/20724da3-1ed3-4e29-b6d4-b4e2b75d0978" width="500"/>

Let me know if that works for you! (I'll move that style setting under a _Positioning_ sub-category in the next release, where it should have been in the first place.)

**Reply by daxmonson** [reply]
That worked! I was fiddling in the style settings plugin positioning menu for your plugin, didn't see there was a specific floating toolbar menu for positioning. Beautiful and well-thought out. Donated.

**Reply by chrisgurney** [reply]
@daxmonson Good to hear, and thank you very much!
